doc/apps/panel.md

```diff
@@ -112,6 +116,33 @@ ploomber-cloud deploy --watch
 To ensure your app doesn't break on re-deployments, pin your [dependencies.](pin-dependencies)
 ```
 
+(customize-deployment)=
+## Customize deployment
```

Comment:
I think we can move this section to the FAQ so we don't repeat ourselves.

the one exception is Shiny, since that's the only R framework we support, we can put this docker thing in the Shiny section

Reply:
[Moved to FAQ](https://github.com/ploomber/doc/pull/196/files#diff-06950dd861443fb8233dd171ce6a68c04fc5e21fac3af4e66e757a1f8ae808d4R108-R118)